packages/runtime/datastore-definitions/src/channel.ts

```diff
@@ -164,6 +164,8 @@ export interface IDeltaHandler {
 	 * resubmission of this message (call it A') should avoid inserting that content, and resubmission of the subsequent op that removed it (B') would
 	 * account for the fact that A' never inserted content.
 	 */
+	// TODO: Use something other than `any` (breaking change)
+	// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
I wonder if these could be made `@sealed` instead so these changes would not be breaking. Not for this PR though.

Reply:
I was wondering the same, but came to the same conclusion that it was out of scope 😋